tools/schema-exporter/schema-exporter.go

```diff
@@ -0,0 +1,52 @@
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+
+	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"kubevirt.io/containerized-data-importer/pkg/operator/resources/cluster"
+	cdioperator "kubevirt.io/containerized-data-importer/pkg/operator/resources/operator"
+	"kubevirt.io/containerized-data-importer/tools/util"
+)
+
+var (
+	exportPath = flag.String("export-path", "", "")
+)
+
+func main() {
```

Comment:
Probably it is easy to reason from code, but just a short description of what it does or why it is needed and what format it exports would be useful.

Reply:
Yes good point, I should put some comments in there about what this does. Its taking the schemas in code, and dumping them out as yaml files. But I should explain that.